build.gradle.kts

```diff
@@ -4,14 +4,19 @@ plugins {
     kotlin("jvm") version "1.4.10"
 }
 
+object Versions {
+    const val ktor = "1.4.0"
```

Comment:
Good idea, we should move all versions used in the script here.

Reply:
done